lib/renderer/override.js

```diff
@@ -100,7 +100,10 @@ window.open = function(url, frameName, features) {
     features = '';
   }
   options = {};
-  ints = ['x', 'y', 'width', 'height', 'min-width', 'max-width', 'min-height', 'max-height', 'zoom-factor'];
+
+  // TODO remove hyphenated options in both of the following arrays for 1.0
+  ints = ['x', 'y', 'width', 'height', 'min-width', 'max-width', 'min-height', 'minHeight', 'max-height', 'maxHeight', 'zoom-factor', 'zoomFactor'];
```

Comment:
@kfranqueiro Yes it should, thanks for mentioning this, will open a new pull request and ping you for review :+1: